packages/teleterm/src/ui/TopBar/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+import Popover from 'design/Popover';
+import { MoreVert, OpenBox, Add } from 'design/Icon';
+import { Box, Text, Flex } from 'design';
+
+import { AccessRequestDocumentState } from 'teleterm/ui/services/workspacesService';
+
+import { useIdentity } from '../Identity/useIdentity';
+
+import { NavigationItem } from './NavigationItem';
+
+export function NavigationMenuContainer() {
```

Comment:
Is this component needed? I'd use early return in `NavigationMenu` if `activeRootCluster` is empty.